### ISSUE TITLE
Update to new ENVs and official builds

### ISSUE THREAD

**Issue by lionkor** [RouHim/beammp-container-image#51]
Hi! As of the upcoming 3.2.2 release, there are multiple linux builds, which you can pull directly from github. These will be named: `BeamMP-Server.$DISTRO.$DISTRO_VERSION.$ARCH`, for example `BeamMP-Server.ubuntu.22.04.x86_64`.

These only depend on `liblua5.3-0` (on ubuntu, as an example). There are also arm builds.

These now support overriding of server config settings via the ENV, for example BEAMMP_PORT, BEAMMP_NAME, BEAMMP_MAP, etc.
These are provided for all settings in the `General` category.

With these changes, this image no longer needs to build the server manually. If you decide to still do that, please also provide debug info files, like the currently latest release does already! :)

**Comment by lionkor** [comment]
To be clear: The benefit of these containers using the official build is that the official build, while stripped of debug info, has associated debug info and should be debuggable as a result.

**Comment by RouHim** [comment]
Yeah, that sounds really good. Compiling via qemu really takes ages :D Thanks for the tips, I'll definitely look into how I can integrate it into my containerfiles. Are there any artefacts that I could test this with? If not, it would be great if you could ping me again when there is a pre-release.

**Comment by lionkor** [comment]
Yes, artifacts here:

[BeamMP-Server.ubuntu.22.04.x86_64.zip](https://github.com/RouHim/beammp-container-image/files/13917105/BeamMP-Server.ubuntu.22.04.x86_64.zip)
[BeamMP-Server.ubuntu.22.04.arm64.zip](https://github.com/RouHim/beammp-container-image/files/13917106/BeamMP-Server.ubuntu.22.04.arm64.zip)
[BeamMP-Server.ubuntu.20.04.x86_64.zip](https://github.com/RouHim/beammp-container-image/files/13917108/BeamMP-Server.ubuntu.20.04.x86_64.zip)
[BeamMP-Server.ubuntu.20.04.arm64.zip](https://github.com/RouHim/beammp-container-image/files/13917110/BeamMP-Server.ubuntu.20.04.arm64.zip)
[BeamMP-Server.debian.12.arm64.zip](https://github.com/RouHim/beammp-container-image/files/13917112/BeamMP-Server.debian.12.arm64.zip)
[BeamMP-Server.debian.12.x86_64.zip](https://github.com/RouHim/beammp-container-image/files/13917113/BeamMP-Server.debian.12.x86_64.zip)
[BeamMP-Server.debian.11.arm64.zip](https://github.com/RouHim/beammp-container-image/files/13917115/BeamMP-Server.debian.11.arm64.zip)
[BeamMP-Server.debian.11.x86_64.zip](https://github.com/RouHim/beammp-container-image/files/13917116/BeamMP-Server.debian.11.x86_64.zip)

The *only* dependency is `liblua5.3-0`.

**Comment by RouHim** [comment]
For the record
```
ldd BeamMP-Server                                                      
	linux-vdso.so.1 (0x00007ffe78f35000)
	liblua5.3.so.0 => not found
	libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x00007f2a1ba00000)
	libm.so.6 => /usr/lib/libm.so.6 (0x00007f2a1bd13000)
	libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x00007f2a1c602000)
	libc.so.6 => /usr/lib/libc.so.6 (0x00007f2a1b81e000)
	/lib64/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x00007f2a1c656000)
```

**Comment by lionkor** [comment]
Yeah, it's a GNU/Linux executable

**Comment by lionkor** [comment]
I'll look into doing a musl build

**Comment by lionkor** [comment]
But I'd prefer if this docker image used official builds, because we have debug info for those, so i can debug crashes ;)

**Comment by RouHim** [comment]
I would also prefer using your builds, that saves a lot of time. Musl builds are optional, would only be practical because I could then use alpine images as a basis, if that is too complicated or requires too much effort, it doesn't matter. What about arm 32 bit builds for older raspberry pi devices? Would that be possible?

**Comment by lionkor** [comment]
Just looked into it; sadly musl-gcc can't build C++ applications, so musl beammp-server won't happen anytime soon (needs more work)

**Comment by lionkor** [comment]
I can look into getting an arm32 VM on one of our systems -- the reason we were able to do arm64 is because hetzner offers nice arm64 servers that we can just use plug&play. I'll look into doing automated arm32 builds via qemu :)

**Comment by RouHim** [comment]
Just an idea, if it's too time-consuming it's probably not worth the effort. But thanks for your support.

**Comment by RouHim** [comment]
@lionkor 

> These now support overriding of server config settings via the ENV, for example BEAMMP_PORT, BEAMMP_NAME, BEAMMP_MAP, etc.

Do you already know the exact names of the env vars?
If so could you please enumerate them?

**Comment by lionkor** [comment]
Just like that, yeah. They are already implemented in the latest release.

BEAMMP_DEBUG
BEAMMP_PRIVATE
BEAMMP_PORT
BEAMMP_MAX_CARS
BEAMMP_MAX_PLAYERS
BEAMMP_MAP
BEAMMP_NAME
BEAMMP_DESCRIPTION
BEAMMP_TAGS
BEAMMP_RESOURCE_FOLDER
BEAMMP_AUTH_KEY
BEAMMP_LOG_CHAT

**Comment by lionkor** [comment]
@RouHim latest release just from today, 3.2.2, has builds
